<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>KDEDocSourceUI</class>
 <widget class="QWidget" name="KDEDocSourceUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>100</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" colspan="2" >
    <widget class="QCheckBox" name="spellCheckBox" >
     <property name="whatsThis" >
      <string>If you select this check box the words from the KDE documentation are spell-checked before they are inserted into the new dictionary.</string>
     </property>
     <property name="text" >
      <string>&amp;Compare to OpenOffice.org dictionary:</string>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="KUrlRequester" name="ooDictURL" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="whatsThis" >
      <string>With this input field you select the OpenOffice.org directory that will be used to spellcheck the words of the new dictionary.</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="KLanguageButton" name="languageButton" />
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="languageLabel" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="whatsThis" >
      <string>With this combo box you select which of the installed languages is used for creating the new dictionary. KMouth will only parse documentation files of this language.</string>
     </property>
     <property name="text" >
      <string>&amp;Language:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>KLanguageButton</class>
   <extends>QWidget</extends>
   <header>klanguagebutton.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>spellCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>ooDictURL</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
